week-4/Homework/mandatory/4-groceries.js

```diff
@@ -29,34 +29,48 @@ Exercise 1:
 */
 // Gather all week item names into this array
 let weeklyGroceriesToBuy = [];
+for (let items in weeklyMealPlan) {
+  weeklyGroceriesToBuy.push(weeklyMealPlan[items]);
+}
 
-
+console.log(weeklyGroceriesToBuy);
 
 /*
 Exercise 2:
-  Loop through your list again, but now only collect the weekend items into the weeklyGroceriesToBuy array.
+  Loop through your list again, but now only collect the weekend items into the weekendGroceriesToBuy array.
   Then use console.log() to print out the list.
 */
 // Gather weekend item names into this array
 let weekendGroceriesToBuy = [];
+for (let items in weeklyMealPlan) {
+  if (items.startsWith("s")) {
+    weekendGroceriesToBuy.push(weeklyMealPlan[items]);
+  }
```

Comment:
similarly here ( think about a way to join arrays)
also do you need to check if it starts with S,?? there is an array with the weekend days, think how can you use it (creating DRY code)

Reply:
Again, used Spread Operator, but I could not find an array with weekend days. 
That s why used weeklyPlan rather than creating another array.